### PR TITLE
🔧 Create an npm package instead of using git submodules

### PR DESCRIPTION
**Pull request by insulineru**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -18,12 +18,12 @@ interface IWalletStoreState {
   address: string | null
   addressShort: string | null
   addressDomain: string | null
-  balance: string | null
+  balance: string | number | null | undefined
```
Comment (iower) [inline src/types.ts]: `string | number | null | undefined` looks inconsistent.
Is it easy to make types at least `balance: string | null`?
(of course, better `balance: number | null` / `balance: BigNumber | null`)

Comment (insulineru) [reply src/types.ts:21]: Yo, if we didn't have these features with `any`, [like this](https://github.com/viaprotocol/web3-wallets/blob/367285760a9358f8f5ce43668de5bde62cb9ff31/src/context/WalletProvider.tsx#L30-L31), there would be a lot less problems :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@viaprotocol/web3-wallets",
+  "private": false,
+  "version": "1.0.1",
+  "author": "Aleksey Lisun<insuline.ru@gmail.com>",
```
Comment (iower) [inline package.json]:
```suggestion
  "author": "Via Protocol (https://via.exchange)",
```

More like collective authorship 😀

File: src/utils/index.ts [hunk 3]
```diff
@@ -58,7 +58,7 @@ export const isValidAddress = async (chainId: number, address: string) => {
   throw new Error(`Not implemented or wrong chainId ${chainId}`)
 }
 
-export const shortenAddress = address => {
+export const shortenAddress = (address: string) => {
   if (typeof address === 'string') {
```
Comment (iower) [inline src/utils/index.ts]: `typeof address` no longer needed (always `string`)

**Comment by iower** [comment]
Epic milestone, no doubt.

File: src/types.ts (hunk 1 above)
Comment (iower) [reply src/types.ts]: Outdated

**Review by iower — APPROVED** [review]

**Pull request merged**